### PR TITLE
#94 Unify examples

### PR DESCRIPTION
**Pull request by anatoly-scherbakov**
# What is done here?

* Comparison table of JSON vs YAML: Added new block — "Features"
* All examples now tell us about the specification itself
* Section about Comments from the FAQ moved to the dedicated section of the spec
* `.yaml` extension added
* Fragment Identifiers section moved to the Addendum about the Extended Profile.

A few fragments of the spec were transcribed into YAML-LD files as examples:

* Comparison of JSON vs YAML,
* List of namespace prefixes,
* Description of the specification itself based on schema.org vocabulary.

***
<a href="https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/pull/100.html" title="Last updated on Jul 1, 2023, 11:41 AM UTC (04287f0)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/100/1863b0e...04287f0.html" title="Last updated on Jul 1, 2023, 11:41 AM UTC (04287f0)">Diff</a>

### CODE REVIEW

**Comment by anatoly-scherbakov** [comment]
@TallTed :vulcan_salute:

File: spec/index.html [hunk 1]
```diff
@@ -963,21 +1047,25 @@ <h3>application/ld+yaml</h3>
...
         <dl>
           <dt>Magic number(s):</dt>
-          <dd>See application/yaml</dd>
+          <dd>See <code>application/yaml</code></dd>
           <dt>File extension(s):</dt>
-          <dd>.yamlld</dd>
+          <dd>
+            <ul>
+              <li><code>.yaml</code></li>
+              <li><code>.yamlld</code></li>
```
Comment (anatoly-scherbakov) [inline spec/index.html:1061]: @ioggstream what do you think of this include of `.yaml` extension?

File: spec/index.html [hunk 2]
```diff
@@ -1214,9 +1160,9 @@ <h2>Best Practices</h2>
 
       <aside class="example" title="Convenience Context">
         <p>Code of the <a>convenience context</a>. Available as:
-          <a href="https://yaml-ld.dev/context">https://yaml-ld.dev/context</a>.
+          <a href="https://github.com/json-ld/convenience-context">https://github.com/json-ld/convenience-context</a>.
```
Comment (anatoly-scherbakov) [inline spec/index.html:1163]: TODO: replace with json-ld.github.io/… 

Where is the context, in fact?

**Comment by anatoly-scherbakov** [comment]
# TODO

* schema.org/ — slash is unnecessary

**Comment by TallTed** [comment]
> * schema.org/ — slash is unnecessary

This needs more context, to know whether the above is accurate or not.

**Comment by anatoly-scherbakov** [comment]
@TallTed here is the change: https://github.com/json-ld/yaml-ld/pull/104

File: spec/index.html (hunk 2 above)
Comment (gkellogg) [reply spec/index.html:1163]: Should be at `https://json-ld.org/contexts/yaml-ld.jsonld` or similar.

**Review by gkellogg — APPROVED** [review]

File: spec/index.html (hunk 2 above)
Comment (anatoly-scherbakov) [reply spec/index.html:1163]: I have submitted a PR to json-ld.org for that.

**Pull request merged**